### DISCUSSION TITLE
it looks like mlp does not interpolate properly public vars for static replicas

### DISCUSSION THREAD

**Discussion by Epado** [mia-platform/community#262] [Q&A]
### Description

Here is the pipeline error:
map[string]interface {}(nil): yaml: invalid map key: map[string]interface {}{"MESSAGING_SERVICE_REPLICAS":""}

**Comment by davidebianchi** [comment]
Hi @Epado!
Can you open the issue in https://github.com/mia-platform/mlp repo? And can you add a repro case? Thank you!
